code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -281,6 +281,54 @@
 	//You don't belong in this world, monster!
 	mytray.reagents.remove_reagent(type, volume)
 
+/datum/reagent/water/salt
+	name = "Saltwater"
+	description = "Water, but salty. Smells like... Central command shuttle docking?"
```

Comment:
```suggestion
	description = "Water, but salty. Smells like... the station infirmary?"
```

i think your current description is way too meta but this one has a nice double meaning